love/src/Utils.js

```diff
@@ -1853,6 +1853,8 @@ export function trimString(string, length = 100) {
 /**
  * Function to parse HTML, generated by react-quill, to Jira Markdown.
  * Check https://jira.atlassian.com/secure/WikiRendererHelpAction.jspa?section=all for more info
+ * This function also parse Jira tickets names that are not hyperlinks.
+ * Allowed projects are in AUTO_HYPERLINK_JIRA_PROJECTS.
  * @param {string} html html to be parsed
```

Comment:
Not related to your PR, but it the double `html` needed here?

Reply:
Thanks and good catch! So the first `html` is the name of the variable itself, although I will adjust the description for better readability. Here are some examples from the oficial docs: https://jsdoc.app/tags-param.html.

Reply:
Thanks for the explanation. Perhaps place a hyphen `-` between both, as suggested by the docs link, so it is clear which is the variable name and where the description starts?